plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/preferences/OpenShiftPreferencePage.java

```diff
@@ -126,7 +63,16 @@ public String getName() {
 			return extensions;
 		};
 
-		public abstract String getLocation();
+		public String getLocation() {
+			if( locationProvider == null ) {
+				 // TODO fill in windows default loc
+				String[] platforms = new String[]{Platform.OS_LINUX, Platform.OS_WIN32};
+				String[] cmds = new String[]{"oc","oc.exe"};
+				String[] defLocs = new String[]{null, "/usr/bin/oc"};
```

Comment:
No.  There is no default location for windows oc tools. In windows, it's just a zip that the user expands wherever they want. For this reason, we have a null default location, which means the path will be searched instead, and, if not found, a 'where oc.exe' will be executed.